highfive/configs/_global.json

```diff
@@ -2,6 +2,12 @@
     "groups": {
         "core": ["@brson", "@nikomatsakis", "@alexcrichton", "@aturon"],
         "crates": ["@alexcrichton"],
-        "doc": ["@steveklabnik"]
+        "doc": ["@steveklabnik", "@GuillaumeGomez"]
+    },
+    "mentions": {
+        "rustdoc-html": {
```

Comment:
I fear this means enough changes to the code changes that it is not worth be reviewing that part yet.